runtime.md

```diff
@@ -54,6 +54,76 @@ The lifecycle describes the timeline of events that happen from when a container
 
 Note: The lifecycle is a WIP and it will evolve as we have more use cases and more information on the viability of a separate create phase.
 
+## Operations
+
+OCI compliant runtimes MUST support the following operations, unless support for the operation can not be supported by the base operating system (kernel) itself.
```

Comment:
I still think @mrunalp's [earlier suggestion][1] for simplifying this sentence would be a good change.

[1]: https://github.com/opencontainers/specs/pull/225#discussion_r42028244